api/tickets.js

```diff
@@ -1,45 +1,57 @@
-import Cookies from 'cookies'
-import { allowCors } from '../lib/corsHelper.js';
-import {getTicketsForOwner, resetOwnerKeys, appendMessage} from '../lib/persistence.js';
-const owner = process.env.OWNER;
-
-
+import Cookies from "cookies";
+import { allowCors } from "../lib/corsHelper.js";
+import {
+  getTicketsForOwner,
+  resetOwnerKeys,
+  appendMessage,
+} from "../lib/persistence.js";
+// const owner = process.env.OWNER;
 
 /**
- * 
- * @param {import('next').NextApiRequest} request 
- * @param {import('next').NextApiResponse} response 
+ *
+ * @param {import('next').NextApiRequest} request
+ * @param {import('next').NextApiResponse} response
  */
-export default allowCors(async function handler(
-    request,
-    response
-) {
-    const key = Buffer.from(process.env.CIPHER_SECRET, 'hex')
-    const cookies = new Cookies(request, response, { secure: true })
-    const selfURL = `https://${request.headers.host}/`
-    const siwe = cookies.get('siwe');
-    const projectKey = request.headers['x-project-key'];
-    const origin = request.headers.origin;
+export default allowCors(async function handler(request, response) {
+  const key = Buffer.from(process.env.CIPHER_SECRET, "hex");
+  const cookies = new Cookies(request, response, { secure: true });
+  const selfURL = `https://${request.headers.host}/`;
+  const siwe = cookies.get("siwe");
+  const projectKey = request.headers["x-project-key"];
+  const origin = request.headers.origin;
 
-    // TODO: get it from cookie, but time is short so hardcode for now
+  // TODO: get it from cookie, but time is short so hardcode for now
 
-    //const test = await getTicketsForOwner('123');
-    if (request.method.toLowerCase() === 'get') {
-        const result = await getTicketsForOwner(owner);
+  //const test = await getTicketsForOwner('123');
+  if (request.method.toLowerCase() === "get") {
+    const owner = request.query?.owner;
+    console.log(owner);
```

Comment:
2 console logs. perhaps an if check if owner is null throw an error?

Reply:
removed the duplicate log.

if check is there, on line 30-33